karma.py

```diff
@@ -47,11 +47,22 @@ def karma(self, event):
     @cmd
     def top_karma(self, event):
         """Get 5 people with most karma points."""
+        args = event.args
         output = ""
         inverted = [(value, key) for (key, value) in self.karmas.items()]
         karmees = sorted(inverted, reverse=True)
-        # Takes top 5 or less if len(karmees) < 5
-        karmees = karmees[:5]
+
+        if len(args) == 0:
```

Comment:
@JakubNvk is there a good reason to treat `len(args) == 0` as a special case? I do not think so -- we could easily set `n` to some sensibe default at the top of this function and then use the code below to try to convert the first argument to a number as you currently do.

Does that sound doable?